Jsontotable clickable links (#40167)
demisto/content: Packs/FiltersAndTransformers/ReleaseNotes/1_2_96.md

#### Scripts

##### JsonToTable
- Updated the Docker image to: *demisto/python3:3.12.8.3296088*.

> - Added support for the _url_keys_ argument to specify a list of keys in a given JSON table that should become clickable links.

Reviewer: ```suggestion
- Added support for the *url_keys* argument to specify a list of keys in a given JSON table that should become clickable links.
```

> 
> - Added support for **url_keys** argument that a list of keys in the given json table that should be turned into a clickable url.

Reviewer: ```suggestion

- Added support for the _url_keys_ argument to specify a list of keys in a given JSON table that should become clickable links.
```
